src/app/shared/launch.service.ts

```diff
@@ -84,11 +84,23 @@ export abstract class LaunchService {
      * @param path The GA4GH Tool's path
      * @param versionName The ToolVersion's name
      */
-    getNxtTestJsonString(workflowPath: string, versionName: string) {
+    getTestJsonString(workflowPath: string, versionName: string, type: string) {
+      let urlType = 'PLAIN_NFL';
+      switch (type) {
+        case 'wdl':
+          urlType = 'PLAIN_WDL';
+          break;
+        case 'cwl':
+          urlType = 'PLAIN_CWL';
+          break;
+        default:
+          urlType = 'PLAIN_NFL';
```

Comment:
Might be better to return a more visible error rather default to nextflow. If we go down the route of adding a fourth language, we might miss this.